nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -147,6 +147,44 @@ $(CsWinRTFilters)
     <Copy SourceFiles="$(CsWinRTGeneratedFilesDir)\$(AssemblyName).winmd" DestinationFolder="$(TargetDir)" UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
   </Target>
 
+  <!-- ====================== -->
+  <!-- Call The GuidPatch tool on the projection .dll -->
+  <Target Name="CsWinRTInvokeGuidPatcher" Condition="'$(CsWinRTEnableGuidPatch)' == 'true'" AfterTargets="Compile" BeforeTargets="Link">
+    <!-- We need to invoke GuidPatch.exe. This lives in two different places depending on our environment. 
+
+         If we are in the CsWinRT repo, we don't have a .nupkg to use to find the exe, 
+           then we need to rely on CsWinRTGuidPatchRepoPath and CsWinRTGuidPatchRuntimePath from src/Directory.Build.props 
+           to get a hold of the files needed 
+           
+         If we are not in the CsWinRT repo, we can use the package paths we know -->
+
+    <PropertyGroup Condition="'$(CsWinRTInRepo)' == 'true'">
+      <!-- We use _DirectoryBuildTargetsBasePath as it is the only way I found to get "c:/.../cswinrt/src" -->
+      <CsWinRTGuidPatchExePath>$(_DirectoryBuildTargetsBasePath)\$(CsWinRTGuidPatchRepoPath)</CsWinRTGuidPatchExePath>
+      <CsWinRTGuidPatchRuntimeDll>$(_DirectoryBuildTargetsBasePath)\$(CsWinRTGuidPatchRuntimePath)</CsWinRTGuidPatchRuntimeDll> 
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(CsWinRTInRepo)' != 'true'">
```

Comment:
We probably don't want to do cswinrt repo checks in a target we ship for public consumption if we don't have to.  Another way to handle this is set value it if is not already set and in the cswinrt repo we can set it ahead of time in directory.props.  so something like:

      <CsWinRTGuidPatchExePath Condition="'$(CsWinRTGuidPatchExePath)'==''">$(CsWinRTPath)build\tools\</CsWinRTGuidPatchExePath>